lib/RTCPeerConnection.js

```diff
@@ -16,6 +16,7 @@ var merge = require('merge'),
 		REGEXP_NORMALIZED_CANDIDATE: new RegExp(/^candidate:/i),
 		REGEXP_FIX_CANDIDATE: new RegExp(/(^a=|\r|\n)/gi),
 		REGEXP_RELAY_CANDIDATE: new RegExp(/ relay /i),
+		REGEXP_HOST_CANDIDATE: / typ host/i,
```

Comment:
I would accept the faster solution regardless which one it is. I expected that creating a `RegExp` instance is the same as creating a literal regexp.